index.js

```diff
@@ -111,10 +115,20 @@ class Installer {
       )
       child.pending = BB.resolve()
       .then(() => extract.child(child.name, child, childPath, this.config))
-      .then(() => readJson(childPath, 'package.json'))
+      .then(() => readPkgJson(path.join(childPath, 'package.json')))
       .then(pkg => {
         return this.runScript('preinstall', pkg, childPath)
         .then(() => this.extractTree(child))
+        .then(() => binLink(pkg, childPath, false, {
+          force: this.config.config.force,
+          ignoreScripts: this.config.lifecycleOpts.ignoreScripts,
+          log: this.log,
+          name: pkg.name,
+          pkgId: pkg.name + '@' + pkg.version,
+          prefix: this.prefix,
```

Comment:
I'm not sure this is right, btw. opts.prefix is used for `man` linking, which is usually like `/usr/local` in npm I think. I should rename this property `manPrefix`. Probably ok to land for now, but we should fix this before launch.

```
function linkMans (pkg, folder, parent, gtop, opts, cb) {
  if (!pkg.man || !gtop || process.platform === 'win32') return cb()

  var manRoot = path.resolve(opts.prefix, 'share', 'man')
  opts.log.verbose('linkMans', 'man files are', pkg.man, 'in', manRoot)
```